apps/e2e/test/test_helper.exs

```diff
@@ -1 +1 @@
-ExUnit.start(exclude: [:e2e, :skip], seed: 0)
+ExUnit.start(exclude: [:e2e, :skip], seed: 0, timeout: 300_000)
```

Comment:
what is the purpose of this timeout? If it is to fix test flakiness, isn't 300,000 a bit much?

Reply:
There is only 1 E2E test. 5 minutes will be well under the time of other integration tests like Andi.

This helps locally to make sure no false failures are happening just because local machine is running slow.